src/random_bq_to_anndata.py

```diff
@@ -0,0 +1,44 @@
+from google.cloud import bigquery
+import argparse
+import random
+import anndata as ad
+
+
+def get_random_ids(project, dataset, client, num_cells):
+    query = client.query(f"SELECT MAX(cas_cell_index) AS max_table_number FROM `{project}.{dataset}.cas_cell_info`")
+    max_cell_id = int([row.max_table_number for row in list(query.result())][0])
+    print(f"Getting {num_cells} random IDs between 0 and {max_cell_id}...")
+    cell_ids = list(range(0, max_cell_id + 1))
```

Comment:
This does assume the cell ids in the db blanket `[0, max_cell_id]` which is true for now, but if at some point that isn't the case we may get fewer results than requested. Might be worth adding a comment to this effect but I wouldn't bother coding for scenario unless that actually happens.

Reply:
added a comment